PixelHunter1995/Player.cs

```diff
@@ -28,6 +27,16 @@ public AnimationTileset AnimationTileset
             set => this.CharComp.AnimationTileset = value;
         }
 
+        public Vector2 FeetPosition
```

Comment:
Feels like something that should be generic to all moveable things. Perhaps PositionComponent should be extended?

Reply:
Fixed, added to CharacterComponent (since we need to know the character's animation tileset).